frontend/src/pages/premium.module.scss

```diff
@@ -8,7 +8,7 @@
   width: $content-max;
   max-width: 100%;
   margin: 0 auto;
-  padding: $spacing-2xl;
+  padding: $spacing-lg;
```

Comment:
Note: this change just brings it in line with `index.page.ts`, where a similar change was applied. Without it, the margin around the phone image on small screens is really big, which now means that there's a big layer of white padding surrounding a relatively thin grey padding around the phone.